### DISCUSSION TITLE
Put logo overtop of navbar

### DISCUSSION THREAD

**Discussion by nn0ble** [tailwindlabs/tailwindcss#5232] [Help] [answered]
I want to make my logo float ontop of my navbar like this image but I cant figure it out.

the logo is a png img

![](https://i.ibb.co/1qy1W9B/Screen-Shot-2021-08-16-at-6-11-41-PM.png)

**Comment by simonswiss** [comment, marked as answer]
Hey!

There are a few ways to do that. If you want to avoid dealing with absolute positioning and arbitrary spacing, you could set a fixed height to your navigation bar, and set it to be a `flex` container with its children centred vertically with `items-center`.

Here's an example where the logo is taking more vertical space than the navbar's height, therefore achieving something similar to your wireframe:

https://play.tailwindcss.com/I8qY5byjfQ

There are probably many other ways to achieve this, but that should get you going 👍

![image](https://user-images.githubusercontent.com/485747/129649644-9fe628bf-427d-4849-a4ba-2dd486beef5c.png)

**Reply by simonswiss** [reply]
And here's an updated version using an image for the logo instead 👍😸

https://play.tailwindcss.com/bMlPaNCP5M

![image](https://user-images.githubusercontent.com/485747/129650156-36a194c0-d058-4173-a0c1-87b818d35622.png)

**Reply by simonswiss** [reply]
If you want the nav elements all centered (instead of using the full width of navbar), you can change the `justify-content` class on the `nav` element to `justify-center` and add a `gap` or some padding on the links 👍

**Reply by nn0ble** [reply]
Thank you, that is exactly what I wanted

**Reply by simonswiss** [reply]
Awesome - glad that was helpful 👍
